lib/osf-components/addon/components/search-page/styles.scss

```diff
@@ -178,23 +178,40 @@
             list-style: none;
             margin-bottom: 0;
             padding-left: 0;
-
             width: 100%;
         }
 
         li {
             display: inline-flex;
             white-space: nowrap;
-            margin-right: 10px;
+            width: calc(100%/6);
+            justify-content: center;
 
             &.active,
             &:hover {
-                background-color: $color-bg-gray;
+                background-color: $color-bg-white;
             }
         }
 
+        li:nth-of-type(1),
+        li:nth-of-type(2),
+        li:nth-of-type(4) {
+            width: calc(100%/6 + 25px);
+        }
+
+        li:nth-of-type(3) {
+            width: calc(100%/6 + 55px);
+        }
+
+        li:nth-of-type(5),
+        li:nth-of-type(6) {
+            width: calc(100%/6 + 10px);
+        }
+
         a {
-            padding-bottom: 10px;
+            padding: 18px 0 10px;
+            width: 100%;
+            text-align: center;
         }
     }
```

Comment:
I am afraid these special casing for each `<li>` is only patching the problem by perpetuating a paradigm that is not quite ideal.
Since the `<ul>` itself is also a flexbox, there seems to be another way to fix this.

Reply:
There are a number of optimizations that can be implemented that will make a better solution. CSS calc may still work but this can be reviewed with the final list of changes. Likely this change will now be included with those either with the hotfix PR or Phase II.